multimedia/fadecandyserver/Makefile

```diff
@@ -21,23 +21,32 @@ define Package/fadecandyserver
 	SECTION:=multimedia
 	CATEGORY:=multimedia
 	TITLE:=FadeCandy Server
-	URL:=https://github.com/scanlime/fadecandy
-	DEPENDS:=+libpthread +librt +uclibcxx +libstdcpp +libc 
+	URL:=https://github.com/bluewavenet/fadecandyserver
+	DEPENDS:=+libpthread +librt +libstdcpp
 endef
 
 define Package/fadecandyserver/description
 	FadeCandy Server For OpenWrt.
```

Comment:
2 spaces instead of tabs in the description as well.

Reply:
Is there a technical reason or is it a set standard?
It works as it is and all looks fine in Packages.manifest.

Reply:
No official one as far as I know. It's consistent with other packages though.